chains/v6/chains_dev.json

```diff
@@ -52,6 +52,11 @@
             {
                 "name": "Sub.ID",
                 "account": "https://sub.id/{address}"
+            },
+            {
+                "name": "Polkaholic",
+                "account": "https://polkaholic.io/account/{address}?group=overview&chainfilters=all",
```

Comment:
Do we need `?group=overview&chainfilters=all`? If it is not really necessary it might save a significant amount of bytes

Reply:
yes, you are right, it can be skipped